src/classical/ultraspherical.jl

```diff
@@ -199,8 +199,17 @@ function \(A::Ultraspherical, B::Jacobi)
     Diagonal(Ã[1,:]./A[1,:]) * (Ã\B)
 end
 function \(A::Jacobi, B::Ultraspherical)
-    B̃ = Jacobi(B)
-    (A\B̃)*Diagonal(B[1,:]./B̃[1,:])
+    if B == Ultraspherical(-1/2) && (A == Jacobi(-1, 0) || A == Jacobi(0, -1))
```

Comment:
Add a comment why this special case is needed.

Also, where in `(A\B̃)*Diagonal(B[1,:]./B̃[1,:])` are the `NaN`s coming in?

Reply:
update the comment to mention even if it is defined the above won't work because of division by zero.